src/client/config/staticData.tsx

```diff
@@ -20,12 +20,11 @@ export const WelcomeNotification: MyNotification = {
         <li>
           hoe u wijzigingen kunt doorgeven als er iets niet klopt;
         </li>
-        <li> informatie over uw buurt op een overzichtelijke <a href="https://mijn.amsterdam.nl/buurt"> kaart </a>.</li>
+        <li> informatie over uw buurt op een overzichtelijke <a href="/buurt"> kaart </a>.</li>
```

Comment:
```suggestion
        <li> informatie over uw buurt op een overzichtelijke <MaRouterLink href={AppRoutes.BUURT}>kaart</MaRouterLink>.</li>
```
Hiermee heb je geen page-reload.